src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -302,14 +302,15 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 			case IntrinsicId.Nullable_GetUnderlyingType:
 			case IntrinsicId.Expression_Property when calledMethod.HasParameterOfType (1, "System.Reflection.MethodInfo"):
 			case var fieldOrPropertyInstrinsic when fieldOrPropertyInstrinsic == IntrinsicId.Expression_Field || fieldOrPropertyInstrinsic == IntrinsicId.Expression_Property:
-			case IntrinsicId.Type_get_BaseType: {
+			case IntrinsicId.Type_get_BaseType:
+			case IntrinsicId.Type_GetConstructor: {
 					var instanceValue = MultiValueLattice.Top;
 					IReadOnlyList<MultiValue> parameterValues = methodParams;
 					if (calledMethodDefinition.HasImplicitThis ()) {
 						instanceValue = methodParams[0];
 						parameterValues = parameterValues.Skip (1).ToImmutableList ();
 					}
-					return handleCallAction.Invoke (calledMethodDefinition, instanceValue, parameterValues, out methodReturnValue);
+					return handleCallAction.Invoke (calledMethodDefinition, instanceValue, parameterValues, out methodReturnValue, out _);
```

Comment:
Would it make sense to always call `handleCallAction.Invoke`, and then handle any non-shared intrinsics, to avoid duplicating the cases above?

Reply:
It does make sense - but I didn't want to do it in this change. That should happen when we have shared all of the intrinsics and those which can't be shared should all switch to the same behavior as the Array.Empty does in this change.
Similarly to MakeGenericMethod - it's not finished yet, and it doesn't handle empty input well yet - again, once all of that is shared we will fix it up.

The change was already getting out of hand as is, I didn't want to make it even bigger.